less/bootstrap.less

```diff
@@ -56,3 +56,4 @@
 @import "spacing.less";
 @import "circle.less";
 @import "logos.less";
+@import "../node_modules/transferwise-iconfont/style.less";
```

Comment:
Could you change the grunt task so that it copies this file to the bootstrap less folder first, perhaps called icons.less? This will need to happen before the less compilation. At the moment the standard grunt task fails, until you do an npm install, if we commit a version to 'less' grunt will work.  We'd need to do an npm install to update the dependency periodically.